src/raw.rs

```diff
@@ -208,81 +240,88 @@ pub const REDISMODULE_HASH_DELETE: *const RedisModuleString = 1 as *const RedisM
 
 #[allow(clippy::not_unsafe_ptr_arg_deref)]
 pub fn call_reply_type(reply: *mut RedisModuleCallReply) -> ReplyType {
```

Comment:
I believe all those functions should be `public(crate)` we do not want the user to use them directly right? I know its a breaking change, maybe this PR should be on the next major version?

Reply:
I was also thinking about that. I was wondering why those were `pub` and visible for everyone, but didn't dare to change: I thought there was a reason it was so. If you think the same as I and there are no reasons, I suggest we do break the compatibility here and make those `pub(crate)` and only expose the Rust API we control, and not these functions.

Reply:
I agree, but we can only do it on a major version.

Reply:
Now, it is hidden behind `pub(crate)` (the whole module).